RELEASE_NOTES.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+ * Resolved processing failures for Illumina runs with incomplete XML configuration files
```

Comment:
Please add "Known Issues" under the 2.3.0 changes and note this issue, so it's clear that that is a bad release